proto/poktroll/application/undelegations.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package poktroll.application;
+
+option go_package = "github.com/pokt-network/poktroll/x/application/types";
+
+import "cosmos_proto/cosmos.proto";
+
+// Undelegation is a message type used to schedule undelegating an application
+// from a gateway.
+message Undelegation {
```

Comment:
Let's update the name. `Undelegation` is VERY general purpose in crypto.

`UndelegationFromAppToGatewayEvent` for example is better.